docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -54,194 +24,171 @@ export COBALT_PIPELINE_NAME="Cobalt CICD Pipeline"
+        1. Select 'Import a repository' from the Repos tab sub-menu and click [Import]
+        1. Enter the Cobalt Clone URL (https://github.com/microsoft/cobalt.git) and select Import
 
-    
-```bash
-az devops configure --defaults organization="$DEFAULT_ORGANIZATION"
-az devops project create --name "$TEMPLATE_DEVOPS_PROJECT_NAME" --source-control git --visibility private
-az devops configure -d project="$TEMPLATE_DEVOPS_PROJECT_NAME"
-```
-
-* Create new repository by fetching source code from the master branch of Cobalt's open-source github project
-    * Select Repos tab within side-navigation menu
-    * Select 'Import a repository' from the Repos tab sub-menu and click [Import]
-    * Enter the Cobalt Clone URL (https://github.com/microsoft/cobalt.git) and select Import
-
-        ![Clone Button](https://user-images.githubusercontent.com/10041279/63459072-8ec4cf00-c419-11e9-8ef4-ee7db827e49c.png)
+            ![Clone Button](https://user-images.githubusercontent.com/10041279/63459072-8ec4cf00-c419-11e9-8ef4-ee7db827e49c.png)
 
     * Rename your repository
-        * Click your Repo Name (ex. Cobalt-Contoso) at the top of the page.
+        1. Click your Repo Name (ex. Cobalt-Contoso) at the top of the page.
 
             ![Repo Name dropdown](https://user-images.githubusercontent.com/10041279/63615290-9d8ebb80-c5aa-11e9-9136-64295640205b.png)
 
-        * Select Manage Repositories
-        * Under Git repositories, find "Cobalt-Contoso" and select the ellipses
-        * Select Rename repository
+        1. Select Manage Repositories
+        1. Under Git repositories, find "Cobalt-Contoso" and select the ellipses
+        1. Select Rename repository
 
             ![Rename Repo dropdown](https://user-images.githubusercontent.com/10041279/63615540-3b828600-c5ab-11e9-9174-07f23b3193bf.png)
 
-        * Give it a new name and click [Rename]
+        1. Give it a new name and click [Rename]
 
             | Naming Recommendation  | Template Repo Strategy |
             |-------------|-----------|
             | Cobalt-Hello-World-Contoso | If the aim is to introduce oneself or the organization to Cobalt, we recommended a name that reflects the spirit of the Azure Hello World Cobalt template. |
             | Cobalt-AZ-ISO-Contoso | If the aim is to have a single repository represent a single Cobalt template, and thereafter, to have one repo per template, we recommend a name that reflects the Cobalt Template being deployed. In this naming example, the name assumes this repo will be dedicated to deploying the Cobalt *az-isolated-service-single-region* template |
             | Cobalt-Contoso | If the aim is to use a single repository as ground truth for all potential patterns across your organization, effectively having to manage a combination of Cobalt patterns from a single repo, it's recommended to stick with a name that matches the project name. |
 
+    * Initialize new Azure Devops pipeline
```

Comment:
More context would be helpful

Reply:
I added a bit more explanation for each naming recommendation.